### ISSUE TITLE
Update Fatigue

### ISSUE THREAD

**Issue by GeoffArmstrong** [Microsoft/AzureStorageExplorer#1058]
I use Storage Explorer on a number of different computers.  It's being actively maintained, and gets many updates (good!).  The install is relatively slow, and it feels like I'm prompted to update it every time I launch the app (bad!).

Since this is a Microsoft project, I was wondering if you guys could walk the walk of providing a version of this tool that can be installed via the Microsoft App Store?  That way I can stop having to click my way through updates on the ~5 machines I use this tool on.

Oh, and the same goes for the Mac App Store.

**Comment by MatthewSteeples** [comment]
Seconded. Installation via the Store means that we can also deploy it via Intune and Windows Store for Business so that our developers don't have to track it down to perform the initial install

**Comment by chryton** [comment]
There is also the issue on the Mac that updates are not self installing but rather opening a zip file in a private app folder and you are expected to just pull it out and replace the app with the newer version. This would lead to confusion, error, and definitely a security issue if a user is working on a restricted system.

**Comment by MRayermannMSFT** [comment]
@chryton , on macOS what would be your ideal update experience? Would you want us to simply replace the .app file for you? Or something else?

**Comment by chryton** [comment]
@MRayermannMSFT Yes, preferably it would replace the .app file for the user. Most apps update in place and do not require any manual input from the user other than possibly dialogue confirmations. A good example of this is the VScode update process.

**Comment by hanssens** [comment]
@MRayermannMSFT I would like to second what @chryton mentions; specifically on macOS the update process downloads a .zip in, what appears to be in a deeply nested (cache?) location. You have to then locate the .zip file, extract it manually and copy it over to for example the Apps folder (overwriting the existing one).

It's not the end of the world but I concur it's quite tedious and, indeed annoying. 

Having a simple automatic replace process process, as seen in Visual Studio Code or basically any other Electron application, that executes this in the background would be really of great value.
